packages/p2p/src/stores/general-store.js

```diff
@@ -739,19 +699,13 @@ export default class GeneralStore extends BaseStore {
             if (!!response && response.error) {
                 floating_rate_store.setApiErrorMessage(response.error.message);
             } else {
-                const {
-                    fixed_rate_adverts,
-                    float_rate_adverts,
-                    float_rate_offset_limit,
-                    fixed_rate_adverts_end_date,
-                    override_exchange_rate,
-                } = response.website_status.p2p_config;
+                const { fixed_rate_adverts, float_rate_adverts, float_rate_offset_limit, fixed_rate_adverts_end_date } =
+                    response.website_status.p2p_config;
                 floating_rate_store.setFixedRateAdvertStatus(fixed_rate_adverts);
                 floating_rate_store.setFloatingRateAdvertStatus(float_rate_adverts);
                 floating_rate_store.setFloatRateOffsetLimit(float_rate_offset_limit);
                 floating_rate_store.setFixedRateAdvertsEndDate(fixed_rate_adverts_end_date || null);
                 floating_rate_store.setApiErrorMessage(null);
-                if (override_exchange_rate) floating_rate_store.setOverrideExchangeRate(override_exchange_rate);
```

Comment:
Why did we remove this?

Reply:
We no more need this,
As I checked with @farzin-deriv the rates are the same, so we can handle it with useexchangeRate hook

Reply:
We are setting the rate from BO if there's no feed available for the currency. I don't think we can remove this. Can we bring this back?

Reply:
The default rate from BO is base on the USD right?

Reply:
and it should be 1 or 0

Reply:
Could u please take a look at this too :pray: https://deriv-group.slack.com/archives/CPC611WA1/p1689129861441949

Reply:
The discussion there is about moving the `website_status.p2p_config` to a new call which will still have `override_exchange_rate`.

Reply:
actually i think we do need to have the `override_exchange_rate` call... can also check this: https://deriv-group.slack.com/archives/CPC611WA1/p1652100727369029

Reply:
@ameerul-deriv @farrah-deriv As I checked with the mobile team and BE, we don't use `override_exchange_rate` in the current version of production, they will create an endpoint in the future for this, so we can ignore this comment for now :pray: